Update CONTRIBUTING.md with instructions on how to load unpackaged build into Chrome (#1196)
themoeway/yomitan: CONTRIBUTING.md

If no arguments are specified, the command is equivalent to `build.bat --all`.

> ### Loading unpacked build into chrome

Reviewer: ```suggestion
### Loading unpacked build into Chrome
```

If no arguments are specified, the command is equivalent to `build.bat --all`.

> ### Loading unpacked build into Chrome

Reviewer 2: ```suggestion
### Loading unpacked build into Chromium browsers
```

> After building, you can load the compiled extension into Chrome

Reviewer 2: ```suggestion
After building, you can load the compiled extension into Chromium browsers.
```

> - Navigate to [extensions settings](chrome://extensions/)

Reviewer 2: ```suggestion
- Navigate to the [extensions page](chrome://extensions/)
```
